Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -962,11 +964,25 @@ static String mutationObserverNotificationString()
     }
 }
 
-static const WebCore::Color textAnnotationHoverColor()
+static const WebCore::Color textAnnotationHoverColor(RefPtr<Element> pluginElement)
 {
     static constexpr auto textAnnotationHoverAlpha = 0.12;
-    static NeverDestroyed color = WebCore::Color::createAndPreserveColorSpace([[[WebCore::CocoaColor systemBlueColor] colorWithAlphaComponent:textAnnotationHoverAlpha] CGColor]);
-    return color;
+
+    static NeverDestroyed color = [element = pluginElement] -> Color {
+        if (!element)
+            return SRGBA<uint8_t> { 0, 122, 255 };
```

Comment:
Also don't need this, since you can always use the singleton.

Reply:
Fixed